db/migrate/20200515161631_create_payment_card.rb

```diff
@@ -0,0 +1,15 @@
+class CreatePaymentCard < ActiveRecord::Migration[6.0]
+  def change
+    create_table :payment_cards do |t|
+      t.timestamps
+
+      t.string :quote_number
+      t.string :proxy_number
+      t.string :card_number
+      t.string :expiration_date
+      t.string :cvv_number
+
+      t.references :aid_application, foreign_key: true, index: true, null: true
+    end
```

Comment:
I don't think we need to have a search view, but I think you should add an index on "Sequence Number" because that's what we'll expect the workers to be typing into our webform. We might also want to add a uniqueness index to Sequence Number and Proxy Number too for maximum paranoia.